hicexplorer/trackPlot.py

```diff
@@ -243,17 +259,17 @@ def parse_tracks(self, tracks_file):
         :param tracks_file: file path containing the track configuration
         :return: array of dictionaries and vlines_file. One dictionary per track
         """
-        from ConfigParser import SafeConfigParser
+        from configparser import ConfigParser
         from ast import literal_eval
-        parser = SafeConfigParser(None, MultiDict)
+        parser = ConfigParser(None, MultiDict)
         parser.readfp(open(tracks_file, 'r'))
 
         tracks_file_path = os.path.dirname(tracks_file)
 
         track_list = []
         for section_name in parser.sections():
             track_options = dict({"section_name": section_name})
-            if section_name.endswith('[spacer]'):
+            if section_name.startswith('[spacer]'):
```

Comment:
This change is wrong as the spacer labels will be changed to make them unique and the result will be `1. [spacer]` ...